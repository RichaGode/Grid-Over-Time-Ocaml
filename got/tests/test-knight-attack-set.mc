int main()
{
  Knave k;
  Knight n; 
  k = new_knave(10, 20);
  n = new_knight(10, 20);
  print(n.attack_strength);
  
  k = attack_knave(n,k);
  print(k.health);
  
  n = set_knight_attack(n, 10);
  print(n.attack_strength);

  k = attack_knave(n,k);
  print(k.health);

  return 0;
}
