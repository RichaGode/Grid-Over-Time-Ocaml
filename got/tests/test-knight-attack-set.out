50
50
10
40